src/authenticationinapp/authenticationinapp.cpp

```diff
@@ -90,7 +90,7 @@ void AuthenticationInApp::signIn() {
   m_listener->signIn();
 }
 
-const QString& AuthenticationInApp::emailAddress() const {
+const QString AuthenticationInApp::emailAddress() const {
```

Comment:
Why?

Reply:
It seems like `VPNAuthInApp.emailAddress()` is `undefined` in QML when it’s returning a reference.

Reply:
Sorry, I change my mind, make it invokable in the header file:
`Q_INVOKABLE const QString& emailAddress() const  { return m_listener->emailAddress(); }`

Reply:
But keep the &